### PR TITLE
File Added : Assigment5 &6

### PR DESCRIPTION
**Pull request by Amit727619**

### CODE REVIEW

File: amit/html/Assigment5/Amit5_3.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang ="en">
+<head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+</head>
```
Comment (Priyankashelke2411) [inline amit/html/Assigment5/Amit5_3.html]: add style tag in head element

File: amit/html/Assigment5/Assigmt56.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="about.css" />
+        <title>Aboutme</title>
+    </head>
+
```
Comment (Priyankashelke2411) [inline amit/html/Assigment5/Assigmt56.html]: remove extra line

File: amit/html/Assigment5/Assigmt5.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+<html>
+    <head>
+        <style>
+            h1{
+                border: 2px solid black;
+                height: 500px;
+                width: 600px;
+                font-style: oblique;
+            }
+
```
Comment (Priyankashelke2411) [inline amit/html/Assigment5/Assigmt5.html]: Remove extra line and give proper indentations

File: amit/html/Assigment5/Amit5_3.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    </header>
+    <section>
+         <h1>about me</h1>    
+              <article></article>
+
+               <p>klfjlj lajflkj ljgk
+               </p> 
+            
+            </article>
+
```
Comment (Priyankashelke2411) [inline amit/html/Assigment5/Amit5_3.html]: remove extra line space .give proper indentations

File: amit/html/Assigment6/Amit6_1.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignmnt6</title>
+        <link rel="stylesheet" href="Amit6_1.css" />
```
Comment (pooja-nagare) [inline amit/html/Assigment6/Amit6_1.html:8]: here it should be href ="Amit6.css"

File: amit/html/Assigment5/Amit5_1.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    background-color:rgb(238, 73, 73);
+    list-style-type: square;
+}
+.moreabout {
+    border: solid;
+    border: 15px 50px;
+    padding: 25px;
+    background-color:rgb(238, 73, 73);
+    list-style-type: square;
+}
```
Comment (shivani-pawbake) [inline amit/html/Assigment5/Amit5_1.css]: add new line

**Review by shivani-pawbake — CHANGES_REQUESTED** [review]

File: amit/html/Assigment6/Amit6_1.html (hunk 5 above)
Comment (pooja-nagare) [inline amit/html/Assigment6/Amit6_1.html]: here it should be href ="Amit6.css"

**Review by pooja-nagare — COMMENTED** [review]
give proper intendation

File: amit/html/Assigment6/Amit6_1.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>  
+        <div class="aboutme"><i> Personal Detailes </i></div>
+        <hr />
+        <label for="first-name">First Name: </label>
+        <input type="text" name="fname" id="first-name" placeholder="First Name" />
+        <br />
+        <br />
+        <label for="last-name">Last Name: </label>
```
Comment (pooja-nagare) [inline amit/html/Assigment6/Amit6_1.html]: make use form tag

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment5/Amit5_1.css [hunk 8]
```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: rgb(6, 45, 58);
+}
+.aboutme {
+    border: solid;
+    border: 15px 50px;
+    padding: 25px;
+    background-color:#ee4949;
```
Comment (KomalShinde813) [inline amit/html/Assigment5/Amit5_1.css]: space required after :

File: amit/html/Assigment5/Amit5_1.css [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    list-style-type:lower-latin;
```
Comment (KomalShinde813) [inline amit/html/Assigment5/Amit5_1.css]: space required after :

File: amit/html/Assigment5/Amit5_1.css [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+}
+.moreabout {
+    border: solid;
+    border: 15px 50px;
+    padding: 25px;
+    background-color:rgb(238, 73, 73);
```
Comment (KomalShinde813) [inline amit/html/Assigment5/Amit5_1.css]: space required after :

File: amit/html/Assigment5/Amit5_1.css [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+    list-style-type: square;
```
Comment (KomalShinde813) [inline amit/html/Assigment5/Amit5_1.css:16]: space required after :

File: amit/html/Assigment5/Amit5_1.html [hunk 12]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta hhtp-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width ,initial-scale=1.0" />
```
Comment (KomalShinde813) [inline amit/html/Assigment5/Amit5_1.html]: remove space which is present before comma and add after comma

File: amit/html/Assigment6/Amit6.css [hunk 13]
```diff
@@ -0,0 +1,6 @@
+.aboutme {
+     background-color:rgb(68, 158, 194);
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Amit6.css]: space required after :

File: amit/html/Assigment6/Amit6.css [hunk 14]
```diff
@@ -0,0 +1,6 @@
...
+     font-size:27px;
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Amit6.css]: space required after :

File: amit/html/Assigment6/Assig6.html [hunk 15]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignmnt6</title>
+    <style>
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Assig6.html]: check indentation

File: amit/html/Assigment6/Assig6.html [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+         <div>horizontal menu
+         <ul>
+             <li><a href="#" > home </li>
+              <li><a href="#">about us</li>
+                <li><a href="#"contactus></li>  
+         </ul>
+         </div>
+         
+         </body>
+         </html>
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Assig6.html]: check indentation and need extra line at the end of file

File: amit/html/Assigment6/Asssigmt61.html [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+        <style>
+          .center {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+          
+          }
+          .aboutme{
+     background-color:skyblue;
+     font-size: 25px;
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Asssigmt61.html:17]: check indentation

File: amit/html/Assigment6/Asssigmt61.html [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+     width: 97%;
+     text-align: right;
+}
+          </style>
+    </head>
+      <body> 
+        <div class="aboutme"><i>Academic Detailes</i></div>
+     <fieldset>
```
Comment (KomalShinde813) [inline amit/html/Assigment6/Asssigmt61.html:25]: check indentation use prettier

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment6/Assig6.html (hunk 15 above)
Comment (Amit727619) [reply amit/html/Assigment6/Assig6.html]: mam i delete this file by mistake i push on git Assig6.html and Assigmt61

**Review by pooja-nagare — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by shivani-pawbake — APPROVED** [review]

**Review by shivani-pawbake — APPROVED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**